src/css/GoldifyApp.css

```diff
@@ -1,6 +1,8 @@
-.goldify-app, .goldify-footer {
+.goldify-app {
   text-align: center;
   width: 100%;
+  min-height: 100%;
+  height: 100%;
```

Comment:
So this works great on mobile, but when i'm on a regular browser and zoom out a bunch (on the solo page, for example), this causes the footer to be below the screen, and I have to scroll to see the footer even if there's no content above the footer. Is there any way to make this always present if there's extra space at the bottom of the screen?